plugins/scanners/dos/src/main/kotlin/DosScanner.kt

```diff
@@ -233,15 +234,25 @@ class DosScanner internal constructor(
     }
 }
 
-private fun Collection<Package>.getDosPurls(provenance: Provenance = UnknownProvenance): List<String> {
+private fun Collection<Package>.getDosPackages(provenance: Provenance = UnknownProvenance): List<PackageInfo> {
```

Comment:
out of scope: Porpose to rename this function to `toPackageInfo()` and also to refactor it so that it converts only a single item.